### PR TITLE
feat: Map display toggle button

### PR DESCRIPTION
**Pull request by alanlr**
and initial context through query parameter

### CODE REVIEW

**Comment by kevbarns** [comment]
Why not try using and styling this component ? [Switch Component](https://chakra-ui.com/docs/components/switch)

File: ui/services/config.js [hunk 1]
```diff
@@ -95,6 +95,13 @@ export const setUseMock = ({ parameterContext }) => {
   }
 }
 
+export const setDisplayMap = ({ parameterContext }) => {
+  let displayMap = getValueFromPath("displayMap")
+  if(displayMap!==null) {
+    parameterContext.setDisplayMap(displayMap==="false"?false:true)
```
Comment (kevbarns) [inline ui/services/config.js]: `displayMap` can never be `null` according to the code, it's initialized to `true` and can only be set to `false`, I don't think the check is necessary, I would have removed the null check and went like :`parameterContext.setDisplayMap(JSON.parse(displayMap))`

**Review by kevbarns — APPROVED** [review]

**Comment by alanlr** [comment]
> Why not try using and styling this component ? [Switch Component](https://chakra-ui.com/docs/components/switch)

Switch component does not accept icon decoration. I lost some time trying to decorate it.

**Pull request merged**